src/main/features/response/views/statement-of-means/check-and-send.njk

```diff
@@ -141,7 +141,7 @@
 {{ row('School costs (include clothing)', expenses.schoolCosts | numeral, '', false) }}
 {{ row('Food and Housekeeping', expenses.foodAndHousekeeping | numeral, '', false) }}
 {{ row('TV and Broadband', expenses.tvAndBroadband | numeral, '', false) }}
-{{ row('Mobile phone', expenses.mobilePhone | numeral, '', false) }}
+{{ row('Phone', expenses.phone | numeral, '', false) }}
```

Comment:
Are you sure this should also be updated?

It's a monthly expense on phone, not phone number. Please ask somebody from UX team

#REQUEST.

Reply:
I had to add this comment to be able to submit my review with request